proto/api/api.proto

```diff
@@ -53,6 +53,9 @@ service APIService {
   // estimate gas for action
   rpc EstimateGasForAction(EstimateGasForActionRequest) returns (EstimateGasForActionResponse) {}
 
+  // estimate gas for action and transfer not sealed
+  rpc EstimateGasForActionNew(EstimateGasForActionRequestNew) returns (EstimateGasForActionResponse) {}
```

Comment:
if this is v2 of `EstimateGasForAction` call it `EstimateGasForActionV2`.
otherwise we need to name the next versionn `EstimateGasForActionNewNew`

Reply:
no problem